tests/test_yask.py

```diff
@@ -358,6 +358,30 @@ def test_capture_vector_temporaries(self):
         assert np.all(v.data == 0.)
         assert np.all(u.data[1] == 5.)
 
+    def test_constants(self):
+        """
+        Check that :class:`Constant` objects are treated correctly.
+        """
+        c = Constant(name='c', value=2.)
+        p = SparseFunction(name='points', nt=1, npoint=1)
+        u = TimeFunction(name='yu4D', shape=(4, 4, 4), dimensions=(x, y, z),
+                         space_order=0)
+        u.data[:] = 0.
+        op = Operator([Eq(u.forward, u + c), Eq(p.indexed[0, 0], 1. + c)],
+                      subs={t.spacing: 1})
+        assert 'run_solution' in str(op)
+        op.apply(yu4D=u, c=c, t=11)
+        # Check YASK did its job and could read constant grids w/o problems
+        assert np.all(u.data[0] == 20.)
+        # Check the Constant could be read correctly even in Devito-land, i.e.,
+        # outside of run_solution
+        assert p.data[0][0] == 3.
+        # Check re-executing with another constant gives the correct result
+        c2 = Constant(name='c', value=5.)
+        op.apply(yu4D=u, c=c2, t=3)
```

Comment:
There are a few different ways to use constants, eg. by supplying explicit values, and/or creating  `Constant` without initial value. All these work though (tested locally), so that is optional.

Reply:
I'll add them in my current local branch , good spot